src/_global_variables.scss

```diff
@@ -35,3 +49,86 @@ $active: $brand2;
 .active {
   color: $active;
 }
+
+
+// mixins
+
+@mixin transition($property:all, $duration:0.3s, $easing:ease-in-out) {
+  transition: $property $duration $easing;
+}
+
```

Comment:
Doesn't look like a number of these mixins are actually used. Can you double-check?

Reply:
They're used in different parts of the Kashti UI, not necessarily the 3 components we've added so far. I just moved them across as one set of global element, as we port over the rest to reach parity they come into use some more.

Happy to remove anything that's not used once all the components are built out.

Reply:
ah, didn't realize it was for future work. all good.

Reply:
I should have elaborated when inserting a big chunk of code 😬